website/src/blogs/2018-10-06-contributing-to-mayactl-cli-in-hacktoberfest-2018.md

```diff
@@ -0,0 +1,180 @@
+
+    mkdir -p $GOPATH/src/github.com/src/openebs
+    cd $GOPATH/src/github.com/src/openebs
+    git clone https://github.com/<your username>/maya.git
+
+These commands will clone maya into your local system.
+
+Once successful, go into the maya project directory using cd maya, and run the make command in the terminal. This will build all the images of the openebs component and other important binaries, along with mayactl.
+
+Once the command is complete, we will be ready with the mayactl binaries and other maya for testing. You can verify this by running docker images.
+
+If the output is similar to the code above, then you are good to go.
+
+## Minikube Setup for Running a Single-Node Cluster
+
+What is Minikube?
+
+Minikube is a tool that enables users to run Kubernetes locally. Minikube runs a single-node Kubernetes cluster inside a VM on your local system (Learn [more](https://kubernetes.io/docs/setup/minikube/)).
+
+To spin up a kubernetes cluster with minikube, you can use the following command:
+
+    sudo minikube start --vm-driver none
+
+You can verify your cluster by using the kubectl command that was previously installed. Running kubectl get nodes should generate an output similar to the one given below.
+
+    NAME STATUS ROLES AGE VERSIONminikube Ready master 26m v1.10.0
+
+## Setting up OpenEBS on Minikube
+
+Once the kubernetes cluster is ready, it’s time to install openebs. Installing openebs on kubernetes is very easy; just apply the openebs-operator to the cluster using the following command:
+
+    kubectl appy -fhttps://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml.
+
+Applying the openebs-operator creates openebs resources in kubernetes. We can obtain the openebs-related resources in the openebs namespace, and we can verify it by typing the following command:
+
+    ashishranjan738@Ashish-PC:~$ kubectl get pods -n openebs
+    
+    NAME READY STATUS RESTARTS AGE
+    
+    cstor-sparse-pool-ni3o-8485bf44cd-fndj6 2/2 Running 0 5m
+    
+    maya-apiserver-674c77bb4c-dhnw5 1/1 Running 0 6m
+    
+    openebs-ndm-rd7hf 1/1 Running 0 6m
+    
+    openebs-provisioner-6d6c9ccf75-s57j8 1/1 Running 2 6m
+    
+    openebs-snapshot-operator-5cdd4ddc46-c2qj5 2/2 Running 0 6m
+
+## Obtaining Access to mayactl
+
+Once all of the pods are in the Running state, and once the maya-apiserver pod is in the Running state, we can access the mayactl binary by accessing the maya-apiserver pod’s shell. We can gain access to mayactl by using the following command:
+
+    kubectl exec -it <maya-api server pod name> -n openebs
+
+(e.g the pod name in my case is maya-apiserver-674c77bb4c-dhnw5). This will open the pod’s terminal over your system terminal. Typing mayactl version should produce something similar to this:
```

Comment:
```suggestion
(e.g the pod name in my case is `maya-apiserver-674c77bb4c-dhnw5`). This will open the pod’s terminal over your system terminal. Typing `mayactl version` should produce something similar to this:
```